corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -174,7 +174,7 @@ hqDefine('analytix/js/hubspot', [
                 var email = $form.find('[name="email"]').val(),
                     firstname = $form.find('[name="firstname"]').val(),
                     lastname = $form.find('[name="lastname"]').val(),
-                    newUrl = document.location.href + '?email=' + email + '&name=' + firstname + '%20' + lastname;
+                    newUrl = document.location.origin + document.location.pathname + '?email=' + email + '&name=' + firstname + '%20' + lastname;
```

Comment:
I'm curious how this is different

Reply:
`document.location.href` also includes query parameters that are present in the URL, 
so when a form is already submitted then the query parameters already have a value set, so when we set a `newUrl` the value of `name` will contain the values of the previously submitted query parameters